statsmodels/tsa/interp/denton.py

```diff
@@ -215,14 +215,14 @@ def dentonm(indicator, benchmark, freq="aq", **kwargs):
 
     # make partitioned matrices
     #TODO: break this out so that we can simplify the linalg?
-    I = zeros((n+m,n+m))
```

Comment:
`I` isnt an identity matrix here.  Is it the standard usage in the literature?

Reply:
It is not standard but given the other notation in this function I strongly suspect they were following the author.